test/e2e/federation-apiserver.go

```diff
@@ -84,8 +84,44 @@ var _ = framework.KubeDescribe("Federation apiserver [Feature:Federation]", func
 			framework.Logf("Verified that zero clusters remain")
 		})
 	})
+	Describe("Admission control", func() {
+		AfterEach(func() {
+			framework.SkipUnlessFederated(f.Client)
+		})
+
+		It("should not be able to create resources if namespace does not exist", func() {
+			framework.SkipUnlessFederated(f.Client)
+
+			// Creating a service in a non-existing namespace should fail.
+			svcNamespace := "federation-admission-test-ns"
```

Comment:
ideally use a unique namespace name (as generated by the framework), so that running the test multiple times against the same cluster (e.g. as the soak tests do), cannot cause contamination between runs.